cartographer/mapping/2d/grid_2d.cc

```diff
@@ -0,0 +1,102 @@
+#include "cartographer/mapping/2d/grid_2d.h"
```

Comment:
add license

Reply:
Added in https://github.com/googlecartographer/cartographer/pull/1046/commits/a9513a6c14bd7ae84093c90a7cfb33d23de97dff